.changeset/eight-zoos-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+HDS-976 Explicitly set aria-selcted to true or false
```

Comment:
```suggestion
HDS-976 Explicitly set aria-selected to true or false
```
Spotted this typo in the changelog. You can add a commit in https://github.com/hashicorp/design-system/pull/682 if you'd like

Reply:
Oops, thanks for spotting that